### DISCUSSION TITLE
Cannot use import in actions.js

### DISCUSSION THREAD

**Discussion by bangdragon** [quasarframework/quasar#9815] [General - Components / Directives / etc]
In the file store/auth/actions.js I try 
import { getAuth, signOut } from "firebase/auth";
but not work.
How to use getAuth and signOut in actions.js
Thanks

**Comment by hawkeye64** [comment]
@bangdragon Please provide more info
Quasar v1 or v2?

If Quasar v1, then 
```js
<script>
import { mapActions } from 'vuex'
.
.
.
methods: {
  ...mapActions('auth', [
    'getAuth,
    'signOut'
  ])
}
```
Now you can access via `this.getAuth` and `this.signOut`

If you are using Quasar v2, then you will need to go look at Vuex docs and see what has changed because there is no longer "this" available with Composition API.

Also, moving this to discussions...

**Reply by bangdragon** [reply]
Thanks for answering, i use q2.
mapActions not work too

**Reply by hawkeye64** [reply]
I suggest looking at the Vuex v4 docs. I have not had a chance to use it with Quasar v2/Vue v3 either.

**Reply by bangdragon** [reply]
I read and tried all the way on the internet but no luck, can't get to work with dynamic registering vuex store module, so mapActions, mapState...

**Reply by metalsadman** [reply]
@bangdragon try 
```js
import firebase from 'firebase/app'
import 'firebase/auth'

someFunction() {
  firebase.auth().signOut()
}
```
https://discord.com/channels/415874313728688138/856517299816497173/858004264264663060
